File: app/src/main/java/com/walhalla/appextractor/activity/string/MvpContract.java
package com.walhalla.appextractor.activity.string;

import androidx.annotation.StringRes;

import java.util.List;

public interface MvpContract {
    interface View {
        void showResourceRawText(StringItem resource, String content);

        void showError(String message);

        void showSuccess(List<StringItem> list);

        void successToast(@StringRes int res, String aaa);

        void errorToast(@StringRes int res, String aaa);

    }
}

File: app/src/main/java/com/walhalla/appextractor/activity/string/MvpContract.kt
package com.walhalla.appextractor.activity.string;

import androidx.annotation.StringRes;

import com.walhalla.appextractor.resources.StringItemViewModel;

import java.util.List;

public interface MvpContract {
    interface View {
        void showResourceRawText(StringItemViewModel resource, String content);

        void showError(String message);

        void showSuccess(List<StringItemViewModel> list);

        void successToast(@StringRes int res, String aaa);

        void errorToast(@StringRes int res, String aaa);

    }
}
